libraries/chain/db_maint.cpp

```diff
@@ -943,33 +943,6 @@ void database::process_bitassets()
    }
 }
 
-/****
- * @brief a one-time data process to correct max_supply
- */
-void process_hf_1465( database& db )
-{
-   const auto head_num = db.head_block_num();
-   wlog( "Processing hard fork core-1465 at block ${n}", ("n",head_num) );
-   // for each market issued asset
-   const auto& asset_idx = db.get_index_type<asset_index>().indices().get<by_type>();
-   for( auto asset_itr = asset_idx.lower_bound(true); asset_itr != asset_idx.end(); ++asset_itr )
-   {
-      const auto& current_asset = *asset_itr;
-      graphene::chain::share_type current_supply = current_asset.dynamic_data(db).current_supply;
-      graphene::chain::share_type max_supply = current_asset.options.max_supply;
-      if (current_supply > max_supply && max_supply != GRAPHENE_MAX_SHARE_SUPPLY)
-      {
-         wlog( "Adjusting max_supply of ${asset} because current_supply (${current_supply}) is greater than ${old}.", 
-               ("asset", current_asset.symbol) 
-               ("current_supply", current_supply.value)
-               ("old", max_supply));
```

Comment:
Do we have any asset with `current_supply > GRAPHENE_MAX_SHARE_SUPPLY`? If yes, we can leave a warning; if no, we can leave a comment somewhere.